.github/workflows/create-executable.yml

```diff
@@ -15,8 +15,8 @@ jobs:
         os: ["windows-latest", "macos-latest"]
```

Comment:
Try splitting it into two workflow files, one for windows and one for mac.

Reply:
Also, when splitting into two workflows, remove continue-on-error: true from the configuration